index.html

```diff
@@ -33,6 +33,293 @@
+      (each containing a track corresponding to the captured
+      <a data-cite="screen-capture#dfn-monitor">monitor</a>).
+      As no [=transient activation=] is required, the <a href="#dfn-user-agent">user agent</a>
+      MUST ensure that appropriate protection, e.g. by allowlisting permitted
+      <a data-cite="url#concept-url-origin">origin</a>.
+    </p>
+  </section>
+  <section id="conformance">
+    <p>
+      This specification defines conformance criteria that apply to a single product: the
+      <dfn data-lt="user-agent">user agent</dfn> that implements the interfaces that it
+      contains.
+    </p>
+    <p>
+      Implementations that use ECMAScript [[ECMA-262]] to implement the APIs defined in this
+      specification must implement them in a manner consistent with the ECMAScript Bindings
+      defined in the Web IDL specification [[!WEBIDL]], as this specification uses that
+      specification and terminology.
+    </p>
+  </section>
+  <section>
+    <h2>
+      Example
+    </h2>
+    <p>
+      The following example demonstrates a request for all
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      using the
+      <code>navigator.mediaDevices.getAllScreenMedia</code>
+      method defined in this document.
+    </p>
+    <pre class="example highlight">
+      try {
+        const mediaStreams = await navigator.mediaDevices.getAllScreenMedia();
+        mediaStreams.forEach((mediaStream, index) => {
+          videoElements[index].srcObject = mediaStream;
+        })
+      } catch (e) {
+        console.log('Unable to acquire screen captures: ' + e);
+      }
+    </pre>
+  </section>
+  <section id="terminology">
+    <h2>
+      Terminology
+    </h2>
+    <p>
+      This document uses the definition of {{MediaStream}}, {{MediaStreamTrack}}
+      from [[!GETUSERMEDIA]], <a data-cite="url#concept-url-origin">origin</a> from [[!url]],
+      {{ScreenDetailed}} from [[!window-placement]],
+      and <a data-cite="screen-capture#dfn-display-surface">monitor</a>
+      from [[!screen-capture]].
+    </p>
+  </section>
+  <section id="capture_multiple_display_surfaces">
+    <h2>
+      Capturing multiple monitors
+    </h2>
+    <p>
+      Capture of all <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      is enabled through the addition of a new
+      {{MediaDevices/getAllScreenMedia}} method on the {{MediaDevices}}
+      interface.
+    </p>
+    <section>
+      <h2>
+        <dfn>MediaDevices</dfn> Additions
+      </h2>
+      <pre class="idl">
+          partial interface MediaDevices {
+            Promise&lt;sequence&lt;MediaStream&gt;&gt; getAllScreenMedia();
+          };
+        </pre>
+      <dl data-link-for="MediaDevices" data-dfn-for="MediaDevices" class="methods">
+        <dt>
+          <dfn>getAllScreenMedia</dfn>
+        </dt>
+        <dd>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
```

Comment:
This looks like it's jumping the gun a bit. Could you move it past the initial validations that result in failure, to where the algorithm actually produces and returns the intended result?

Reply:
moved below the algorithm.